openssl-sys/build.rs

```diff
@@ -323,8 +323,14 @@ fn validate_headers(include_dirs: &[PathBuf]) -> Version {
 #include <openssl/opensslv.h>
 #include <openssl/opensslconf.h>
 
-#if LIBRESSL_VERSION_NUMBER >= 0x20701000
+#if LIBRESSL_VERSION_NUMBER >= 0x20800000
 RUST_LIBRESSL_NEW
+#elif LIBRESSL_VERSION_NUMBER >= 0x20703000
```

Comment:
Given that it doesn't appear that there were any ABI changes from 2.7.0 to 2.7.2, I think we can just merge all of the 2.7.x checks together into one.